helper/geojsonify_meta_data.js

```diff
@@ -8,7 +8,14 @@ var Document = require('pelias-model').Document;
  */
 function addMetaData(src, dst) {
   dst.id = src._id;
-  dst.gid = makeGid(src);
+
+  // at this time the gid's of interpolated addresses are not usable in the place endpoint
+  // to avoid confusion they will be removed altogether for now and replaced in the future when we have determined
+  // the right format for these types of results
+  if (src.source !== 'mixed') {
```

Comment:
interestingly the middleware code has no logic to handle the case of `TIGER` data, instead it is marked as `mixed`, this is not technically correct but it's satisfactory because it would also be impossible to retrieve this content via `/v1/place`.